src/api/routes/address.ts

```diff
@@ -185,6 +186,38 @@ export function createAddressRouter(db: DataStore, chainId: ChainID): RouterWith
     res.json(response);
   });
 
+  router.getAsync('/:stx_address/:tx_id/transactions_with_transfers', async (req, res) => {
```

Comment:
Since this is for a singular transaction, should this be renamed to `transaction_with_transfers`, or even just `transfers` because it's already in the context of a transaction?

Reply:
I have changed the endpoint to `with_transfers` as suggested in the comment below the respective issue:
https://github.com/blockstack/stacks-blockchain-api/issues/622#issuecomment-857064925